recipes/pymapd/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pymapd" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "a636142ff2f18ab3450b81a7d1eae3bed6d25d26e8ba193f508b99091c063026" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - setuptools_scm
+  run:
+    - python
+    - setuptools
+    - six
+    - thrift
+    - typing
```

Comment:
`typing` seems to be a py2 only requirement according to the `setup.py`. If this is the case, please add the appropriate preprocessor selector:
https://conda.io/docs/building/meta-yaml.html#preprocessing-selectors